worker-pools.yml

```diff
@@ -1584,9 +1584,9 @@ pools:
             maxTaskRunTime: 2592000
             enableInteractive: true
       minCapacity: 0
-      maxCapacity: 96
+      maxCapacity: 97
       implementation: generic-worker/worker-runner-linux
-      regions: [us-central1, us-west1, us-east1]
+      regions: [us-central1, us-west1, us-east1, europe-west4]
```

Comment:
Should the new worker be in a dedicated pool to be able to 1) specifically target it, 2) set maxCapacity separately from the other regions?

Reply:
Putting in a separate worker pool wouldn't make it practical to use, unfortunately. We need to try it out a real training (that uses full sized datasets) to get a good idea of how useful it is. We've got enough going on in these pools that it'll end up getting used.

(I would love to be able to set capacities by region, but I'm not sure _that's_ easily doable.)